nyaa/search.py

```diff
@@ -3,23 +3,115 @@
 import math
 import json
 import shlex
+from datetime import datetime, timedelta
 
 from nyaa import app, db
-from nyaa import models
+from nyaa import models, torrents
 
 import sqlalchemy_fulltext.modes as FullTextMode
 from sqlalchemy_fulltext import FullTextSearch
 from elasticsearch import Elasticsearch
 from elasticsearch_dsl import Search, Q
 
+class ElasticTorrentStatsAccessor:    
```

Comment:
What's the purpose behind having an `ElasticTorrentStatsAccessor` class, as its properties can be accessed from the `ElasticTorrentAccessor` directly (particularly with my recommended changes to `__getattr__`)